lib/jwt/jwk/ec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module JWT
+  module JWK
+    class EC
+      extend Forwardable
+      def_delegators :@keypair, :private?, :public_key
+
+      attr_reader :keypair
+
+      KTY    = 'EC'.freeze
```

Comment:
You've got the `frozen_string_literal` pragma in this file, so the `.freeze` is redundant.

```suggestion
      KTY    = 'EC'
```